pom.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.fponce</groupId>
+    <artifactId>hotel-ns</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+
+    <properties>
+        <springframework.version>5.0.8.RELEASE</springframework.version>
+        <springjdbc.version>4.2.7.RELEASE</springjdbc.version>
+        <liquibase-core.version>3.6.2</liquibase-core.version>
+        <spring-boot-web-starter.version>2.0.4.RELEASE</spring-boot-web-starter.version>
+        <postgres-sql.version>42.2.4</postgres-sql.version>
+        <validation-api.version>2.0.1.Final</validation-api.version>
+        <jackson-annotations.version>2.9.3</jackson-annotations.version>
+    </properties>
+
+
+    <modules>
+        <module>hotel-api-model</module>
+        <module>hotel-api-persistence</module>
+        <module>hotel-api-services</module>
+        <module>hotel-api-server</module>
+    </modules>
+
+    <name>hotel-ns</name>
+    <description>Demo project for hotel administration</description>
+
+    <dependencyManagement>
+        <dependencies>
```

Comment:
I recommend use Maven dependency-management feature.

Reply:
Yes, I implemented it using maven dependency management feature for spring boot.